packages/@aws-cdk/pipelines/lib/actions/deploy-cdk-stack-action.ts

```diff
@@ -216,6 +216,11 @@ export class DeployCdkStackAction implements codepipeline.IAction {
    */
   public readonly stackArtifactId?: string;
 
+  /**
+   * Action role
+   */
+  public readonly actionRole: iam.IRole;
```

Comment:
You only actually need the account here, so why not only publish the account? Let's not make the interface wider than it needs to be.